databroker/src/broker.rs

```diff
@@ -1648,7 +1649,13 @@ impl<'a, 'b> AuthorizedAccess<'a, 'b> {
             .await
             .add_change_subscription(subscription);
 
-        let stream = ReceiverStream::new(receiver);
+        let stream = BroadcastStream::new(receiver).filter_map(|result| match result {
+            Ok(message) => Some(message),
+            Err(err) => {
+                debug!("Lagged entries: {}", err);
+                None
+            }
+        });
```

Comment:
This should be forwarded to to the subscriber (of the `broadcast` channel), in order for them to be able to act on it.

Reply:
Don't know if I fully get that, you mean transmit the lagged error as tonic error to the consumer application?

Reply:
I'm saying:
> This should be forwarded to to the subscriber (of the broadcast channel),

i.e. to whoever called subscribe internally in databroker and is subscribing to this stream. That means the decision of what to do when it lags is located in the implementation of a specific service which can then decide whether to report that back over GRPC for example.

Reply:
No way found to propagate the lagged error to the `subscribe` caller, just found a way to pipe the error into the stream and return it back to the end user as tonic error.
The error is received while reading the `receiver`, no idea hot to stop the stream there and return an error to the `subscribe` caller, maybe a different function to 'filter_map' and 'map` can stop the stream and return an error?

Reply:
I think you simply need to change the returned stream from `impl Stream<Item = EntryUpdates>` to `impl Stream<Item = Result<EntryUpdates>>` or something like that..

But perhaps it makes sense to postpone the implementation of that as we currently don't have the need to propagate the fact that a subscriber is lagging to the subscriber (i.e. the GRPC / VISS subscriber).

Reply:
I can try your suggestion, anyway, have been trying hard to trigger the lagged error wit lazy subscriber but not way to produce it, never received on client side any lagged error, no idea why, maybe you can test it as well and see if you can get the corresponding error.